assets/js/langRedirector.js

```diff
@@ -0,0 +1,45 @@
+// Wrap all this to not pollute global namespace
+{
+    // Generate an array of languages from page
+    // TODO: is there a way to do it static using Jekyll?
+    const JDM_LANGS = [...document.querySelectorAll('#language-dropdown > ul > li')].map(el => el.classList[0]);
```

Comment:
Do you know @PyroSamurai or @Nightfirecat ?
I don't off the top of my head, but I can try to find out

Reply:
Given that we can list out all languages in our markup (to generate the languages which appear in the dropdown), yes it is possible. That said, it would need to exist on the page as a global variable to be accessible here. I don't see that as a bad thing so it'd be fine to do so imo